.gitignore

```diff
@@ -1,3 +1,6 @@
 .idea
 .vscode
-packit.iml
\ No newline at end of file
+.token
+packit.iml
+app/test-results
+app/playwright-report
```

Comment:
Would these be useful to upload as artefacts of the CI?

eg. https://github.com/mrc-ide/packit/blob/377f8e487a894825f04a579f51e25d5e557f6e40/.github/workflows/backend-test-and-build.yml#L72-L81

Reply:
Note that `auth.json` is written to the `test-results` dir, so if that is erroneously not deleted (e.g. if playwright dies mid-run), we'd be uploading the github access token for all to see. So `auth.json` might want to be written somewhere else instead.

Reply:
It wouldn't be the github access token, it would just be the temporary packit token, which would only apply to the localhost instance we're running on CI, so I think we should be ok unless we start running tests against prod on CI.  I'll try to omit it auth.json though.